src/map/pc.cpp

```diff
@@ -13493,8 +13495,8 @@ void JobDatabase::loadingFinished() {
 				}
 			}
 
-			// Summoner
-			if( ( class_ & MAPID_BASEMASK ) == MAPID_SUMMONER ){
+			// Summoner / Spirit Handler
+			if( ( class_ & MAPID_BASEMASK ) == MAPID_SUMMONER || ( class_ & MAPID_BASEMASK ) == MAPID_SPIRIT_HANDLER ){
```

Comment:
This and all below should be changed to ( class_ & MAPID_SUMMONER ) == MAPID_SUMMONER and leave out the basemask check. Also applies to the clif.cpp change.